### PR TITLE
fix warning C4717: 'format<std::vector<std::any,std::allocatorstd::an…

### PR DESCRIPTION
**Pull request by shelllet**
…y > >': recursive on all control paths, function will cause runtime stack overflow

### CODE REVIEW

File: Foundation/include/Poco/Format.h [hunk 1]
```diff
@@ -27,7 +27,7 @@
 namespace Poco {
 
 
-std::string Foundation_API format(const std::string& fmt, const Any& value);
+	std::string Foundation_API format(const std::string& fmt, const Any& value);
```
Comment (aleks-f) [inline Foundation/include/Poco/Format.h:30]: why this indentation?

**Review by aleks-f — CHANGES_REQUESTED** [review]
please revert the indentation

File: Foundation/include/Poco/Format.h (hunk 1 above)
Comment (shelllet) [reply Foundation/include/Poco/Format.h:30]: this is the result of vscode auto-formatting

**Review by aleks-f — APPROVED** [review]

**Comment by aleks-f** [comment]
this breaks g++, will be reverted and redone
https://github.com/pocoproject/poco/pull/3732

**Comment by shelllet** [comment]
I don't have any experience with g++, anyone has an idea?

**Comment by aleks-f** [comment]
There's no need for any more ideas regarding this, it has bern taken care of in 1.12.2 release

**Pull request merged**